content/docs/tasks/traffic-management/circuit-breaking/index.md

```diff
@@ -83,160 +95,181 @@ Let's set up a scenario to demonstrate the circuit-breaking capabilities of Isti
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-23:51:51 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
-Ended after 71.05365ms : 30 calls. qps=422.22
-Aggregated Function Time : count 30 avg 0.0053360199 +/- 0.004219 min 0.000487853 max 0.018906468 sum 0.160080597
-# range, mid point, percentile, count
->= 0.000487853 <= 0.001 , 0.000743926 , 10.00, 3
-> 0.001 <= 0.002 , 0.0015 , 30.00, 6
-> 0.002 <= 0.003 , 0.0025 , 33.33, 1
-> 0.003 <= 0.004 , 0.0035 , 40.00, 2
-> 0.004 <= 0.005 , 0.0045 , 46.67, 2
-> 0.005 <= 0.006 , 0.0055 , 60.00, 4
-> 0.006 <= 0.007 , 0.0065 , 73.33, 4
-> 0.007 <= 0.008 , 0.0075 , 80.00, 2
-> 0.008 <= 0.009 , 0.0085 , 86.67, 2
-> 0.009 <= 0.01 , 0.0095 , 93.33, 2
-> 0.014 <= 0.016 , 0.015 , 96.67, 1
-> 0.018 <= 0.0189065 , 0.0184532 , 100.00, 1
-# target 50% 0.00525
-# target 75% 0.00725
-# target 99% 0.0186345
-# target 99.9% 0.0188793
-Code 200 : 19 (63.3 %)
-Code 503 : 11 (36.7 %)
-Response Header Sizes : count 30 avg 145.73333 +/- 110.9 min 0 max 231 sum 4372
-Response Body/Total Sizes : count 30 avg 507.13333 +/- 220.8 min 217 max 676 sum 15214
-All done 30 calls (plus 0 warmup) 5.336 ms avg, 422.2 qps
-{{< /text >}}
-
-Now we start to see the circuit breaking behavior we expect.
-
-{{< text plain >}}
-Code 200 : 19 (63.3 %)
-Code 503 : 11 (36.7 %)
-{{< /text >}}
-
-Only 63.3% of the requests made it through and the rest were trapped by circuit breaking. We can query the istio-proxy stats to see more:
-
-{{< text bash >}}
-$ kubectl exec -it $FORTIO_POD  -c istio-proxy  -- sh -c 'curl localhost:15000/stats' | grep httpbin | grep pending
-cluster.outbound|80||httpbin.springistio.svc.cluster.local.upstream_rq_pending_active: 0
-cluster.outbound|80||httpbin.springistio.svc.cluster.local.upstream_rq_pending_failure_eject: 0
-cluster.outbound|80||httpbin.springistio.svc.cluster.local.upstream_rq_pending_overflow: 12
-cluster.outbound|80||httpbin.springistio.svc.cluster.local.upstream_rq_pending_total: 39
-{{< /text >}}
-
-We see `12` for the `upstream_rq_pending_overflow` value which means `12` calls so far have been flagged for circuit breaking.
+## Adding a client
+
+Create a client to send traffic to the `httpbin` service. The client is
+a simple load-testing client called [fortio](https://github.com/istio/fortio).
+Fortio lets you control the number of connections, concurrency, and
+delays for outgoing HTTP calls. You will use this client to "trip" the circuit breaker
+policies you set in the `DestinationRule`. 
+
+1. Inject the client with the istio sidecar proxy so network interactions are
```

Comment:
i'd rework this documentation to just assume automatic injection is used.  The more options people have to think about, the less likely they are to complete the documentation.

IOW delete lines 106-112

Reply:
I'm happy to remove the sidecar mentions here. Is this something we should systematically root out of the whole 1.0 doc set? I.e., make sure the other writers working on the docs look out for this? I know it always makes me stop and think "I don't know, do I have it enabled or not"?